skietbaan_fe/src/bootstrap/NavbarMenuUserStyles.css

```diff
@@ -1,172 +1,203 @@
-@media only screen and (min-width: 580px) {
-  .dropdown-menu.show {
-    background-color: #f3f4f9;
+@media only screen and (min-width: 576px){
+  .dropdown-menu.show{
+    background-color: #F3F4F9;
     border: none;
-    border: 1px solid #f3f4f9;
+    border: 1px solid  #F3F4F9;
+    
   }
 
-  .leaderboard-icon-grey {
-    height: 40px;
-    width: 59px;
+  .profile-icon-grey{
+    height: 27px;
+    width: 28px;
   }
 
-  .add-score-icon-grey {
-    height: 40px;
-    width: 49px;
+  .leaderboard-icon-grey{
+    height: 26px;
+    width: 26px;
   }
 
-  .create-icon-grey {
-    height: 40px;
-    width: 31px;
+  .add-score-icon-grey{
+    height: 26px;
+    width: 26px;
   }
 
-  .columns-more {
-    display: none;
-    background-image: url("/components/navbar-icons/more");
+  .create-icon-grey{
+    height: 26px;
+    width: 26px;
+  }
+
+  .columns-more{
+    display:none;
   }
 
-  .more-icon-grey {
+  .more-icon-grey{
     height: 38px;
     width: 28px;
     display: none;
     width: 0%;
   }
 
-  .profile-icon-grey {
-    height: 40px;
-    width: 30px;
+  .profile-icon-grey-v2{
+    height: 27px;
+    width: 28px;
   }
 
-  .docs-icon-grey-v2 {
+  .notifications-icon-grey{
+    height: 26px;
+    width: 23px;
+  }
+  
+  .docs-icon-grey-v2{
     height: 40px;
     width: 54px;
   }
 
-  .notifications-icon-grey-v2 {
-    height: 40px;
-    width: 57px;
+  .notifications-icon-grey-v2{
+    height: 26px;
+    width: 23px;
   }
 
-  .second-row-navbar {
+  .second-row-navbar{
     height: 68px;
     padding-top: 15px;
     padding-bottom: 15px;
     width: 100%;
     display: none;
   }
 
-  .columns-v2 {
-    width: 12%;
+  .columns-v2{
+    width: 25%;
   }
 
-  .columns {
-    width: 12%;
+  .columns{
+    width: 20%;
   }
 
-  .first-row-navbar {
-    height: 68px;
+  .first-row-navbar{
+    height: 56px;
     padding-top: 15px;
     padding-bottom: 15px;
     width: 100%;
   }
+  
+  .navbar-admin{
+    background-color: #000000;
+    width: 66%;
+    text-align: center;
+    margin-left: 17%;
+    top: 0;
+    z-index: 100;
+    position: fixed;
+    border-bottom-left-radius: 20px;
+    border-bottom-right-radius: 20px;
+    border-top-left-radius: 0px;
+    border-top-right-radius: 0px;
+  }
 
-  .navbar-admin {
-    background-color: #f3f4f9;
-    width: 100%;
+  .nav-content{
     text-align: center;
   }
 }
 
-@media only screen and (max-width: 579px) {
-  .dropdown-menu.show {
-    background-color: #f3f4f9;
+@media only screen and (max-width:575px){
+  .dropdown-menu.show{
+    background-color: #F3F4F9;
     border: none;
-    border: 1px solid #f3f4f9;
+    border: 1px solid  #F3F4F9;
   }
 
-  .leaderboard-icon-grey {
-    height: 40px;
-    width: 59px;
+  .leaderboard-icon-grey{
```

Comment:
`.leaderboard-icon-grey` and 
`.add-score-icon-grey` and 
`.create-icon-grey` are the same

Reply:
made a generic one